app/routes/umd_tree_cover_loss/raster_tiles.py

```diff
@@ -56,7 +56,41 @@ async def umd_tree_cover_loss_raster_tile(
     background_tasks: BackgroundTasks,
 ) -> Response:
     """
-    UMD tree cover loss raster tile.
+    UMD Tree Cover Loss raster tile.
+
+    There are two types of TCL tiles available here: True color tiles and encoded tiles.
+    The true color tiles have been pre-rendered to appear the same way TCL appears on
+    globalforestwatch.org (i.e. the tiles show pink pixels to indicate tree cover loss).
+    The caveat with the true color tiles is that they do not contain information about
+    the year of loss. This means that they cannot be filtered after the fact (for example
+    to support a dynamic time slider). Whatever time period the true color tiles span is
+    baked-in (either the entire time period the alerts cover, or optionally bound by the
+    start_year and/or the end_year query parameters).
+
+    The encoded tiles, on the other hand, contain the year information but are not ready
+    to be displayed as-is. They can/need to be interpreted and rendered according to
+    the needs of the consumer of the data.
+
+    Note that both types of tiles are thresholded by Tree Cover Density, which means
+    that one needs to know the TCD percentage which one wants to consider necessary to
+    indicate tree cover loss of interest. A small example: If one wants data which
+    indicates loss in areas that have a tree cover density of 75% or more, one would
+    specify a TCD of 75. The allowable thresholds can be seen below in the query
+    parameters.
+    So if you want
+
+    True color tiles (i.e. pink pixels as they appear on the map at globalforestwatch.org), thresholded by tree cover density
+        Specify the style query parameter
+        For example: https://tiles.globalforestwatch.org/umd_tree_cover_loss/v1.10/dynamic/1/1/0.png?style=tcd_10
```

Comment:
Sorry, didn't notice this before, but for true color tiles, why do you sometimes specify the tree cover density by "style=tcd_10" and sometimes by "tcd=10".  Are these two ways of specifying TCD entirely equivalent, or do you have use one or the other depending on if there are other query parameters?

Reply:
As far as I can tell they are equivalent. It SEEMS to me as though one way allows you to specify a start and end date, and the other, well, just doesn't. I know it seems redundant. :/